vector/src/main/java/im/vector/app/features/location/live/map/LocationLiveMapViewFragment.kt

```diff
@@ -94,6 +96,12 @@ class LocationLiveMapViewFragment @Inject constructor() : VectorBaseFragment<Fra
     private fun setupMap() {
         val mapFragment = getOrCreateSupportMapFragment()
         mapFragment.getMapAsync { mapboxMap ->
+            mapboxMap.uiSettings.apply {
+                // Place copyright above the user list bottom sheet
+                setLogoMargins(dimensionConverter.dpToPx(8), 0, 0, dimensionConverter.dpToPx(208))
+                setAttributionMargins(dimensionConverter.dpToPx(96), 0, 0, dimensionConverter.dpToPx(208))
```

Comment:
Could you use the size of the view that these values ​​depend on instead of using static values ?

Reply:
Good idea, done. Although it is not possible to calculate the width of the "mapbox" logo.